README.markdown

```diff
@@ -173,11 +173,14 @@ This documentation assumes there's a `yokozuna` bucket type created and activate
 ``` ruby
 # Create a client and bucket.
 client = Riak::Client.new
-bucket = client.bucket 'pizzas'
+bucket_type = client.bucket_type 'yokozuna'
+bucket = bucket_type.bucket 'pizzas'
 
 # Create an index and add it to a typed bucket. Setting the index on the bucket
 # may fail until the index creation has propagated.
-client.create_search_index 'pizzas'
+index = Riak::Search::Index.new 'pizzas'
+index.exist? #=> false
+index.create!
 client.set_bucket_props bucket, {search_index: 'pizzas'}, 'yokozuna'
```

Comment:
I've been dithering about this for most of a year (c.f. #148 ), so I should probably make a decision about it soon :P